bases/rsptx/interactives/runestone/mchoice/js/mchoice.js

```diff
@@ -154,7 +157,9 @@ export default class MultipleChoice extends RunestoneBase {
             onsubmit: "return false;",
         });
         // Add fieldset and legend for accessibility
-        this.optsFieldSet = document.createElement("fieldset")
+        this.optsFieldSet = document.createElement("fieldset");
+        this.optsFieldSet.setAttribute("role", "radiogroup");
```

Comment:
Some multiple choice questions can have multiple answers in which case it is not a radio group but a bunch of check boxes.  Do you maybe want to check `this.multipleanswers` before setting the role to radiogroup?

Reply:
I see that they still look and work fine in both cases so maybe nothing to worry about.